installers/linux/deb/minikube_0.8-0/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: minikube
+Version: 0.8-0
```

Comment:
As the version comes from DEB_VERSION and the text prefix is "Version: ", most env variables will be equally as long.  I made the regex a variable to enhance readability:
	sed -E -i 's/Version: '$(VERSION_REGEX)'/Version: '$(DEB_VERSION)'/g' installers/linux/deb/minikube_$(DEB_VERSION)/DEBIAN/control

Please let me know if this what you had in mind, thanks!